### PR TITLE
fix: css for obc credentials copy to clipboard

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
fixes: https://issues.redhat.com/browse/DFMS-208

after:
image

before:
image

### CODE REVIEW

File: packages/utils/generics/copy-to-clipboard.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  --pf-c-button--BorderRadius: 0;
+  --pf-c-button--hover--BorderWidth: 1px;
+  --pf-c-button--LineHeight: 20px;
+}
+
+.pf-c-clipboard-copy__group-copy {
+  position: absolute;
+  right: 0;
+  top: 3px;
+}
```
Comment (SanjalKatiyar) [inline packages/utils/generics/copy-to-clipboard.scss:32]: do we have to copy pf classes as well ? did u try without coping this class once, if it is working as expected ?

Comment (SanjalKatiyar) [reply packages/utils/generics/copy-to-clipboard.scss:32]: cause we use pf as a dependency, it should be fine IMO... if it is not a pf class then let's change it's name to just `clipboard-copy__group-copy`

File: packages/utils/generics/copy-to-clipboard.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  color: var(--pf-global--Color--100);
+  display: block;
+  font-family: var(--pf-c-code-block__pre--FontFamily);
+  font-size: var(--pf-c-code-block__pre--FontSize);
+  overflow-wrap: break-word;
+  padding: var(--pf-global--spacer--md);
+  white-space: pre-wrap;
+}
+
+.co-copy-to-clipboard__text {
```
Comment (SanjalKatiyar) [inline packages/utils/generics/copy-to-clipboard.scss:12]:
```suggestion
.copy-to-clipboard__text {
```

File: packages/utils/generics/copy-to-clipboard.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  padding-right: 50px;
+}
+
+.co-copy-to-clipboard__btn,
+.co-copy-to-clipboard__btn:hover {
```
Comment (SanjalKatiyar) [inline packages/utils/generics/copy-to-clipboard.scss:17]:
```suggestion
.copy-to-clipboard__btn,
.copy-to-clipboard__btn:hover {
```

File: packages/utils/generics/copy-to-clipboard.scss (hunk 1 above)
Comment (debjyoti-pandit) [reply packages/utils/generics/copy-to-clipboard.scss:32]: yeah I have tried, without it, it looks like below 

image

Comment (SanjalKatiyar) [reply packages/utils/generics/copy-to-clipboard.scss:32]: okay then change the name of class as mentioned above...

Comment (debjyoti-pandit) [reply packages/utils/generics/copy-to-clipboard.scss:32]: Done

**Pull request merged**